runtime/drivers/duckdb/migrate.go

```diff
@@ -78,9 +85,9 @@ func (c *connection) Migrate(ctx context.Context) (err error) {
 	return nil
 }
 
-func (c *connection) migrateSingle(ctx context.Context, name string, sql []byte, version int) (err error) {
+func (c *connection) migrateSingle(conn *sqlx.Conn, ctx context.Context, name string, sql []byte, version int) (err error) {
```

Comment:
`conn` should come after `ctx` in the param list

Reply:
done